modules/node_modules/@ncigdc/containers/RepositoryPage.js

```diff
@@ -37,8 +38,20 @@ export type TProps = {
 };
 
 export const RepositoryPageComponent = (props: TProps) => {
-  const setAutocompleteCases = value => props.relay.setVariables({ idAutocompleteCases: value, runAutocompleteCases: !!value });
-  const setAutocompleteFiles = value => props.relay.setVariables({ idAutocompleteFiles: value, runAutocompleteFiles: !!value });
+  const setAutocompleteCases = value => props.relay.setVariables(
+    {
+      idAutocomplete: value,
+      autocompleteTypes: !!value ? ['case'] : [],
```

Comment:
if it's always `['case']` then the file autocomplete will also return cases - I think I'm not understanding the question